package.json

```diff
@@ -10,7 +10,7 @@
   ],
   "license": "MIT",
   "scripts": {
-    "dev": "parcel index.html",
+    "dev": "parcel index.html --open",
```

Comment:
Strictly speaking, should be in separate PR. Not a big deal here.

Reply:
👍🏻 Makes sense.